ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -657,6 +660,22 @@ iters = bifoldMap (:[]) (const [])
 flrs :: Bitraversable t => t it flr -> [flr]
 flrs = bifoldMap (const []) (:[])
 
+{-------------------------------------------------------------------------------
```

Comment:
Is GHC 9.2 more sensitive about how the TH splices delineate the module's declarations now?

Reply:
Yes.